### ISSUE TITLE
loosen up ordinances to allow chickens

### ISSUE THREAD

**Issue by whit537** [Ambridge/Ambridge#44]
Ticketing from [Facebook](https://www.facebook.com/groups/324204731118103/permalink/355906167947959/):

> this [video of a kid hugging a chicken](https://www.facebook.com/video.php?v=10152383047076942) could be happening all over Ambridge, (along with the delicious organic local eggs we could harvest).

**Comment by whit537** [comment]
+1 from @JessWhit and I.

**Comment by sbdavies** [comment]
I agree - I think that we should loosen up the ordinances re: animals in general. Not only are animals like chickens prohibited, but you can only have two household pets, no matter what: http://www.ecode360.com/15267676. Obviously, you don't want them to be much looser, but two seems pretty restrictive. I don't really see an issue with someone having a little chicken coop in their yard, either (for hens, obviously, not roosters).

**Comment by sbdavies** [comment]
Looks like the Borough Council is considering this, Chad: http://www.ambridgeconnection.com/local-news/ambridge-council-seeks-input-on-consideration-to-allow-for-urban-agriculture-in-the-borough

**Comment by whit537** [comment]
Thanks @sbdavies! I've added my 2¢ and would encourage others to do the same! :-)

🐔 🐝 🐰

**Comment by whit537** [comment]
I guess there's a meeting going on right now at Bridgetown about this. Maybe @JessWhit can report?
